app-modules/billing/src/Barte/BarteAdapter.php

```
@@ -166,4 +166,34 @@ public function getBillingPortalUrl(User|Company $billable, string $returnUrl, a
 
         return UserBillingManagePage::getUrl();
     }
+
+    public function purchaseCredits(User|Company $billable, Company $company, int $quantity, string $successUrl, string $cancelUrl): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$successUrl'. (undefined)

(UnusedFormalParameter)